Makefile

```diff
@@ -0,0 +1,52 @@
+BIN     = $$(npm bin)
+BABEL   = $(BIN)/babel
+KARMA   = $(BIN)/karma
+WEBPACK = $(BIN)/webpack
+
+.PHONY: clean test test-watch build package.json javascript release example javascript-min
+
+build:
+	@make clean
```

Comment:
`@` before a command silences it, I'm using this to give cleaner build generation logging